week-1-practise/level-1/1-syntax-error-1.js

```diff
@@ -1,14 +1,16 @@
 // There is a syntax error in this code. Fix it.
 
-function addNumbers(a b c) {
-    return a + b + c;
+function addNumbers(a, b, c) {
+  return a + b + c;
 }
 
 var result = addNumbers(1, 3, 4);
 console.log(result); // Expected result 8 - Run the program and make sure it works
 
-
 // Answer these questions:
 // 1. How many parameters does the function take?
+// The above function takes three parameters.
 // 2. What is the function name?
-// 3. Where is the function called? with what parameters?
\ No newline at end of file
+// Functiin name is addNumbers
+// 3. Where is the function called? with what parameters?
+// Function is called on line 7 var result = addNumbers(1,3,4)
```

Comment:
Good!
 // 3. Where is the function called? with what parameters?
The parameters passed to the function call are 1, 3, 4 (actually, these are more correctly called the arguments)

Reply:
got it